examples/nextjs/src/utils/server-side-fetch.ts

```diff
@@ -1,10 +1,9 @@
 const serverSideFetch = async (query: string) =>
-  await fetch(process.env.GRAFBASE_API_URL as string, {
+  await fetch(process.env.GRAFBASE_API_URL!, {
     method: 'POST',
-    // @ts-ignore
     headers: {
       'Content-Type': 'application/json',
-      'x-api-key': process.env.GRAFBASE_API_KEY
+      'x-api-key': process.env.GRAFBASE_API_KEY!
```

Comment:
How will this work for local dev?

Reply:
It's not needed in local dev AFAIK

Reply:
Yes, you don't need it, just set `GRAFBASE_API_URL=http://localhost:4000` in .env file

Reply:
The examples will set the API key to an empty string via the env file, e.g. https://github.com/grafbase/grafbase/blob/main/examples/nextjs/.env.example#L1

So this should work OOTB.